infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSwitchController.kt

```diff
@@ -125,7 +130,7 @@ class LayoutSwitchController(
 
     @PreAuthorize(AUTH_ALL_READ)
     @GetMapping("/{id}/change-times")
-    fun getSwitchChangeTimes(@PathVariable("id") switchId: IntId<TrackLayoutSwitch>): ChangeTimes {
+    fun getSwitchChangeTimes(@PathVariable("id") switchId: IntId<TrackLayoutSwitch>): DraftableChangeInfo {
```

Comment:
Ja täällä jne. En varmaan merkkaa näitä kohtia tästä eteenpäin spämmin välttämiseksi, mutta idea säilyy tästä eteenpäinkin

Reply:
Juu, tuossa ne suurimmaksi osaksi olikin. Osa controllereista olikin mutta kävin nyt läpi ja korjasin kaikki.